sdk/servicebus/ci.yml

```diff
@@ -8,7 +8,8 @@ trigger:
     - release/*
   paths:
     include:
-    - sdk/servicebus/
+    - sdk/servicebus/ci.yml
+    - sdk/servicebus/Azure.Messaging.ServiceBus
```

Comment:
Because there is no current `Azure.ResourceManager.ServiceBus` project, I'm not listing it here.  The `Artifacts` section gets cranky if something doesn't exist and it seems more correct to leave it off entirely than have it half wired up.   Open to suggestion here, if someone feels differently.

Reply:
I don't think you can do anything here ahead of a library showing up. It will just have to be included at the same time the artifact is added.